app/main.py

```diff
@@ -1 +1,37 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive: list = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+
+        self.alive.append(self)
+
+    def __repr__(self) -> str:
+        return ", ".join([
```

Comment:
Write `__repr__` without using `join`, there is no benefit for declaring a list and iterating through it here. Write the f-string in a whole inside `()` here to avoid line breaks.